src/config/jest.config.js

```diff
@@ -15,10 +15,10 @@ const ignores = ['/node_modules/', '/fixtures/', '/__tests__/helpers/', '__mocks
 const jestConfig = {
 	roots: [fromRoot('src')],
 	testEnvironment: ifAnyDep(['webpack', 'rollup', 'react'], 'jsdom', 'node'),
-	collectCoverageFrom: ['src/**/*.js'],
-	testMatch: ['**/__tests__/**/*.js', '**/*.spec.js'],
+	collectCoverageFrom: ['src/**/*.[jt]s?(x)'],
```

Comment:
should it be `[jt]sx?`, this doesn't match _js, ts_ and matches _tx, jx_

Reply:
same below

Reply:
It's the weird glob/extglob syntax. Seems to work afaict, but I'll double check.

https://github.com/micromatch/micromatch#extglobs